update(UI overview): add bonita apps (#1362)
bonitasoft/bonita-doc: md/special-users.md

# Special users

> This page explains the two/three ? other ways to log into Bonita Portal than with a profile: Tenant Administrator (Technical user), Platform Administrator, and System Administrator.

Reviewer: ```suggestion
This page explains the three other ways to log into Bonita than with a regular profile: Tenant Administrator (Technical user), Platform Administrator, and Guest user.
```

I remove portal as platformAdmin cannot really log in on the portal

## Sytem administrator
Quick definition, rights and duties.
Default username password. Where to change it.

> 

Reviewer: ```suggestion
```
There is no such type of account in Bonita portal

## Platform administrator. 

> Quick definition, rights and duties. 

Reviewer: ```suggestion
The platform is the part of the system that is common to all tenants. The platform administrator can manage [through the API](platform-api.md) the platform level resources (including starting/stopping all services of the platform and managing tenants). He can also access the license information (for Enterprise editions).  
```

> Default username password. Where to change it.

Reviewer: ```suggestion
Its username and password are defined in a configuration file and need to be changed for any other usage of a Bonita Tomcat bundle than local development (see
[Bonita Tomcat bundle installation procedure](tomcat-bundle.md#toc3) for more information on how to proceed.
Its default credentials are:
* username: `platformAdmin`
* password: `platform`

```

## Technical user

> Quick definition, rights and duties. 

Reviewer: ```suggestion
```

> Each tenant has an administrator (also known as the **tenant technical user**) with a tenant-specific username and password.

Reviewer: ```suggestion
Each tenant has an administrator (also known as the **tenant technical user**) with a tenant-specific username and password. The tenant administrator can start and stop the BPM services for the tenant, install a Business Data Model, install the resources and the organization.  
```

> * Username default value is `install`
> * password default value is `install`
> Where to change it.

Reviewer: ```suggestion
Their username and password are defined in a configuration file and need to be changed for any other usage of a Bonita Tomcat bundle than local development (see
[Tenant admin credentials page](tenant_admin_credentials.md) for more information on how to proceed.
Its default credentials are:
* username: `install`
* password: `install`
```

> If you modify these values before the first start of the platform, the new values will be the default values for all tenants that will be
> created in the future, including the default tenant.
> 

Reviewer: ```suggestion
```

## Guest user

> Quick definition and rights.
> Link to guest user for how it works.

Reviewer: ```suggestion
With enterprise editions, it is possible to allow public access to some living applications (without the need to sign in to Bonita).
When accessing a public application without any active session on Bonita platform the user will be automatically logged in with a guest user account.  
To know more about this feature, go to the [Guest user access documentation](guest-user.md).
```
